<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cam_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>cam_on</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cam_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>651</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>650</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>650</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>CamControl()</slot>
  <slot>register()</slot>
 </slots>
</ui>
